ddtrace/profiling/collector/memalloc.py

```diff
@@ -75,7 +74,18 @@ def stop(self):
                 pass
             super(MemoryCollector, self).stop()
 
+    def _get_thread_id_ignore_list(self):
```

Comment:
```suggestion
    def _get_ignore_thread_id_set(self):
```

Reply:
It's a list… like a metaphorical list… not a type-list… c'mon 😆

Reply:
but `list` and `set` are both built-in types 🙂  I'd find it confusing calling a `_list` function that's promising a `list` and instead gives me a `set` 😝

Reply:
... how about `_iterable`? 😁